docs/examples/frameworks/jax/jax-multigpu_example.ipynb

```diff
@@ -0,0 +1,304 @@
+{
```

Comment:
<blockquote>Does it mean it will then proceed to the next shard? If they process only items belonging to a particular shard</blockquote><p>This is controlled by <code>stick_to_shard</code> argument. By default it is false, so in the next epoch pipeline will move to the next shard. I added a sentence with the information about this argument. </p>

Reply:
<p>Rest done</p>